cmd/ltops/db.go

```diff
@@ -30,20 +30,23 @@ var dbCommand = &cobra.Command{
 			return errors.Wrap(err, "couldn't load cluster")
 		}
 
-		settings, err := cluster.DBSettings()
-		if err != nil {
-			return errors.Wrap(err, "failed to get database settings")
-		}
-
-		// TODO: support psql
-		return mysql(settings)
+		return shell(&cluster)
 	},
 }
 
-func mysql(settings *ltops.DBSettings) error {
-	logrus.Infof("Connecting to %s:%d", settings.Endpoint, settings.Port)
+func shell(cluster *ltops.Cluster) error {
+	settings, err := (*cluster).DBSettings()
+	if err != nil {
+		return errors.Wrap(err, "failed to get database settings")
+	}
 
-	cmd := exec.Command("mysql", "-u", settings.Username, fmt.Sprintf("-p%s", settings.Password), "-h", settings.Endpoint, "-P", strconv.Itoa(settings.Port), settings.Database)
+	logrus.Infof("Connecting to %s:%d", settings.Endpoint, settings.Port)
+	var cmd *exec.Cmd
+	if (*cluster).Configuration().DBEngineType != "aurora-postgresql" {
```

Comment:
nit: this condition is inverted, making the flow awkward. Might we switch on the potential `DBEngineType` values instead?